frontend/packages/console-app/src/components/quick-starts/controller/QuickStartIntroduction.tsx

```diff
@@ -16,23 +16,29 @@ const QuickStartIntroduction: React.FC<QuickStartIntroductionProps> = ({
   introduction,
   allTaskStatuses,
   onTaskSelect,
-}) => (
-  <>
-    <SyncMarkdownView content={introduction} />
-    <p style={{ marginBottom: 'var(--pf-global--spacer--md)' }}>
-      In this quick start, you will complete {pluralize(tasks.length, 'task')}:
-    </p>
-    {tasks.map((task, index) => (
-      <TaskHeader
-        key={task.title}
-        title={task.title}
-        taskIndex={index + 1}
-        size="md"
-        taskStatus={allTaskStatuses[index]}
-        onTaskSelect={onTaskSelect}
-      />
-    ))}
-  </>
-);
+}) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <SyncMarkdownView content={introduction} />
+      <p style={{ marginBottom: 'var(--pf-global--spacer--md)' }}>
+        {t('quickstart~In this quick start, you will complete {{count}} task', {
```

Comment:
should we use number formatter here as we have `count`

```suggestion
        {t('quickstart~In this quick start, you will complete {{count, number}} task', {
```